monitoring/src/kieker/monitoring/probe/aspectj/operationExecution/OperationStartData.java

```diff
@@ -16,22 +16,17 @@
 
 package kieker.monitoring.probe.aspectj.operationExecution;
 
-final class OperationStartData {
```

Comment:
Why is the class no longer final? Final classes can be executed faster. Esp. for data classes there was an advantage in the past.

Reply:
Ah ok. I understand below, you extend the class. So this might be necessary.